src/index.js

```diff
@@ -1,5 +1,6 @@
 import {OpenForm} from './sdk';
 import './index.scss';
+// import displayComponents from './custom-display-components';
```

Comment:
```suggestion

```

Reply:
it's there deliberately - disabled by default, but easily enabled by uncommenting the commented lines